### PR TITLE
First pass definitions-as-landing-pages

### PR DESCRIPTION
**Pull request by RacheltheEditor**
Removing second person and direct references to the body of the Guide, filling in defs that assumed contextual position within a Guide (this will be a much more detailed and complex project, but easy ones here), moving anything out of scope for a landing page definition into a bullet point.

These are copy edits for production purposes, so both labels.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -502,7 +502,8 @@ is created, and thereafter will increase as new shares are added (issued) and gr
 people in exchange for money or services.
 Outstanding shares may increase or decrease for other reasons too (such as stock splits
 and share buybacks, which we won’t get into here).
-Later, we discuss [several subtleties](#counting-shares) in how shares are counted.
+
+- Later, we discuss [several subtleties](#counting-shares) in how shares are counted.
```
Comment (jlevy) [inline README.md]: Remove bullet? Seems like it's a regular paragraph/sentence.

File: README.md [hunk 2]
```diff
@@ -613,9 +614,11 @@ company’s profit to shareholders, authorized by the board of directors.
 Established public companies and some private companies pay dividends, but this is rare
 among startups and companies focused on rapid growth, since they often wish to re-invest
 their profits into expanding the business, rather than paying that money back to
-shareholders. For example, Amazon
-[has never](https://www.fool.com/investing/2017/12/28/will-amazon-start-paying-a-dividend-in-2018.aspx)
-paid dividends.
+shareholders.
+
+- Amazon, for example,
```
Comment (jlevy) [inline README.md]: Same here. Either top-level elaborative paragraph, or (as discussed) also think it's OK in this case just to keep it part of the definition since it's such an important example.

File: README.md [hunk 3]
```diff
@@ -1070,8 +1076,7 @@ details:
   when the value of the company doesn’t reach the dollar amount investors put into it.
   Because of liquidation preference, those holding preferred stock (investors) will have to
   be paid before those holding common stock (employees).
-  If investors have put millions of dollars into your company and it’s sold, your equity as
-  an employee
+  If investors have put millions of dollars into a company and it’s sold, employees' equity
```
Comment (jlevy) [inline README.md]: oriented quote

File: README.md [hunk 4]
```diff
@@ -1587,11 +1593,13 @@ biggest chunk of tax they pay.
...
-The Social Security wage withholding rate is **6.2%** up to the FICA wage base.
-The Medicare component is **1.45%**, and it does not phase out above the FICA wage base.
-You’ll also hear these called [**payroll taxes**](https://en.wikipedia.org/wiki/Payroll_tax) as
-they often show up on your pay stub.
+that are withheld from a person’s paycheck.
+Employment taxes are also referred to as
+[**payroll taxes**](https://en.wikipedia.org/wiki/Payroll_tax) as they often show up on employee
+pay stubs.
+
+- The Social Security wage withholding rate in 2018 is **6.2%** up to the FICA wage base.
```
Comment (jlevy) [inline README.md]: Let's include these in the definition. Also unbold the numbers per @Hopeopolis 's request.

File: README.md [hunk 5]
```diff
@@ -1600,10 +1608,9 @@ Long-term capital gains tax is often lower than ordinary income tax.
 Many investors hold assets for longer than a year in order to qualify for the lesser tax
 burden of long-term capital gains.
 
-🄳 **Alternative minimum tax (AMT)** is an
-[entirely separate kind of tax](https://en.wikipedia.org/wiki/Alternative_minimum_tax) that is
-neither ordinary income tax nor employment tax, and has its own rules and only applies in
-some situations.
+🄳 **Alternative minimum tax (AMT)** is neither ordinary income tax nor employment tax, but
```
Comment (jlevy) [inline README.md]: Let's wrap this and the next paragraph together:

🄳 **Alternative minimum tax (AMT)** is a
[supplemental income tax](https://en.wikipedia.org/wiki/Alternative_minimum_tax) that applies to certain individuals and in some situations. This type of tax does not come up for many taxpayers, but higher income earners and people in special situations may have to pay large AMT bills. AMT was first enacted in 1982 is not the same as ordinary income tax or employment tax.

File: README.md [hunk 6]
```diff
@@ -1822,13 +1829,14 @@ regarding stock awards and stock options.
   If it’s been years since the stock was first granted and the company is now worth a lot,
   the taxes owed could be quite significant.
 - 🄳 The Internal Revenue Code, in
-  [Section 83(b)](https://www.law.cornell.edu/uscode/text/26/83), offers an alternative, called
+  [Section 83(b)](https://www.law.cornell.edu/uscode/text/26/83), offers protection to anyone
```
Comment (jlevy) [inline README.md]: ... offers taxpayers receiving equity in exchange for work the option to pay taxes early, before they vest, and thereby avoiding taxes at time of vesting. If qualified, a person can tell the IRS they prefer this alternative in a process called an [**83(b) election**](http://acceleratedvesting.com/what-is-an-83b-election-and-when-do-i-make-it-part-1-with-graphic/). Paying taxes early with an 83(b) election can potentially reduce taxes significantly. If the shares go up in value, the taxes owed at vesting might be far greater than the taxes owed at the time of receipt.

(found the previous wording a bit hard to parse)

File: README.md [hunk 7]
```diff
@@ -2405,20 +2413,20 @@ can add to a company, so that they are fully prepared to negotiate for a better
...
-decision for your personal situation.
+🄳 When making a job offer, companies will often give a candidate a **verbal offer** first,
+to speed things along and facilitate the negotiation, following it with a **written offer**
+if it seems like the candidate and the company are close to agreement on the terms of the
+offer. The written offer takes the form of an 📥[**offer letter**](https://www.upcounsel.com/employee-offer-letter),
+which is just the summary sent to the candidate, typically with an expiration date and
+other details and [paperwork](#documents-and-agreements).
+
+- If you are ready to accept the terms of the offer letter, you can go ahead and sign.
+- Although companies often want you to sign right away to save time and effort, if you’re
```
Comment (jlevy) [inline README.md]: Elide second bullet?

File: README.md (hunk 1 above)
Comment (RacheltheEditor) [reply README.md]: Fixed.

File: README.md (hunk 5 above)
Comment (RacheltheEditor) [reply README.md]: 1979*

Comment (RacheltheEditor) [reply README.md]: Revised. Improvement, expansion still needed.

File: README.md (hunk 6 above)
Comment (RacheltheEditor) [reply README.md]: Clear change. Revised slightly for further clarity.

File: README.md [hunk 8]
```diff
@@ -1002,13 +1003,16 @@ There are some key subtleties you’re likely to come across in the way
...
+- 🄳 **Issued and outstanding** refers to the number of shares actually issued by a company to
+  shareholders, and does not include shares that others may have an option to purchase.
+- 🄳 **Fully diluted** refers to all of the shares that a company has issued, all of the
+  shares that have been set aside in a stock incentive plan, and all of the shares that
+  could be issued if all convertible securities (such as outstanding warrants) were
+  exercised.
+
+A key difference between fully diluted shares and shares issued and outstanding is that
+the total of fully diluted shares will include all the shares in the employee option pool
+that are reserved for but *not yet issued to* employees.
```
Comment (jlevy) [inline README.md]: remove extraneous "for"

File: README.md [hunk 9]
```diff
@@ -1822,13 +1829,12 @@ regarding stock awards and stock options.
...
+  [Section 83(b)](https://www.law.cornell.edu/uscode/text/26/83), offers taxpayers receiving
+  equity in exchange for work the option to pay taxes on their options before they vest.
+  If qualified, a person can tell the IRS they prefer this alternative in a process called
   an
-  [**83(b) election**](http://acceleratedvesting.com/what-is-an-83b-election-and-when-do-i-make-it-part-1-with-graphic/),
-  that protects a person from high potential tax at time of vesting, ensuring they’ll be
-  taxed on the receipt of the property (the stock), rather than at the time the stock vests.
-  With an 83(b) election, you’re telling the IRS you want to pay taxes early, and this can
-  potentially reduce your tax significantly:
+  [83(b) election](http://acceleratedvesting.com/what-is-an-83b-election-and-when-do-i-make-it-part-1-with-graphic/).
```
Comment (jlevy) [inline README.md]: boldface is lost; restore to make a definition

File: README.md [hunk 10]
```diff
@@ -1600,12 +1604,15 @@ Long-term capital gains tax is often lower than ordinary income tax.
...
-special situations often have to pay very large AMT bills.
+🄳 **Alternative minimum tax (AMT)** is a
+[supplemental income tax](https://en.wikipedia.org/wiki/Alternative_minimum_tax) that applies
+to certain individuals in some situations.
+This type of tax does not come up for many taxpayers, but higher income earners and people
+in special situations may have to pay large AMT bills.
+AMT was first enacted in [1979](https://www.taxpolicycenter.org/briefing-book/what-amt) in
+response to reports that 155 wealthy individuals had paid no income tax in 1966. It is
+[calculated differently](https://www.nerdwallet.com/blog/taxes/alternative-minimum-tax-amt/)
+than the standard income tax.
```
Comment (jlevy) [inline README.md]: Pls preserve the original distinctions from ordinary income tax and employment tax (terms already defined).

It is not the same as ordinary income tax or employment tax, and is
calculated [according to its own rules](https://www.nerdwallet.com/blog/taxes/alternative-minimum-tax-amt/).

File: README.md [hunk 11]
```diff
@@ -1600,12 +1604,15 @@ Long-term capital gains tax is often lower than ordinary income tax.
...
-neither ordinary income tax nor employment tax, and has its own rules and only applies in
-some situations.
-This type of tax does not come up for many people, but higher income earners and people in
-special situations often have to pay very large AMT bills.
+🄳 **Alternative minimum tax (AMT)** is a
+[supplemental income tax](https://en.wikipedia.org/wiki/Alternative_minimum_tax) that applies
+to certain individuals in some situations.
+This type of tax does not come up for many taxpayers, but higher income earners and people
+in special situations may have to pay large AMT bills.
+AMT was first enacted in [1979](https://www.taxpolicycenter.org/briefing-book/what-amt) in
```
Comment (jlevy) [inline README.md]: Nit on link styling: Anchor text should be "in 1979" as it is about the event, not the year.

Comment (jlevy) [reply README.md]: (Or the verb or verb phrase.)

File: README.md (hunk 8 above)
Comment (RacheltheEditor) [reply README.md]: I wouldn't consider that extraneous. They are reserved for employees, but not yet issues to them. No?

Comment (jlevy) [reply README.md]: Yes, technically, but I'm not sure it adds clarity (I actually thought it was a typo). The relevant point in *this* context is that the pool is reserved, not that it's for employees. Also, the unused parts of the pool are not guaranteed to go to employees; it can get reallocated later, a point that probably should be mentioned in the pool definition. https://www.alleywatch.com/2014/01/vc-101-what-happens-to-the-employee-option-pool-after-an-acquisition/

**Comment by jlevy** [comment]
LGTM!
